sdk/core/azure-core/src/main/java/module-info.java

```diff
@@ -28,6 +28,10 @@
     exports com.azure.core.util.serializer;
     exports com.azure.core.util.tracing;
     exports com.azure.core.cryptography;
+    exports com.azure.core.implementation.serializer;
+    exports com.azure.core.implementation.serializer.jsonwrapper;
+    exports com.azure.core.implementation.serializer.jsonwrapper.jacksonwrapper;
+    exports com.azure.core.implementation.serializer.jsonwrapper.api;
```

Comment:
Generally we don't want to be exporting implementation packages, what is the exact need for this being exported?

Reply:
jsonwrapper is used by azure-search test and samples to demonstrate converting search results to fully typed documents 
@JonathanGiles

Reply:
Alan's observation is important here - our expectation is that we never export 'implementation' packages as public API. If there is code in here that end users need to use, then it should be considered in the wrong package and proposed to go into a new package outside of implementation.

Reply:
@JonathanGiles this can be done - jsonwrapper utility will be moved a few levels up under util,serializer package as part of this PR

Reply:
@JonathanGiles @alzimmermsft the jsonwrapper util was moved outside of implemetation and module-info updated.